graphite-ch-optimizer.go

```diff
@@ -165,11 +164,11 @@ func processFlags() error {
 	// ClickHouse set
 	fc := pflag.NewFlagSet("clickhouse", 0)
 	fc.StringP("server-dsn", "s", viper.GetString("clickhouse.server-dsn"), "DSN to connect to ClickHouse server")
-	fc.Duration("optimize-interval", viper.GetDuration("clickhouse.optimize-interval"), "The active partitions won't be optimized more than once per this interval, seconds")
+	fc.Duration("optimize-interval", viper.GetDuration("clickhouse.optimize-interval"), "The partition will be merged unless it have active writes for more than, duration")
 	// Daemon set
 	fd := pflag.NewFlagSet("daemon", 0)
 	fd.Bool("one-shot", viper.GetBool("daemon.one-shot"), "Program will make only one optimization instead of working in the loop (true if dry-run)")
-	fd.Duration("loop-interval", viper.GetDuration("daemon.loop-interval"), "Daemon will check if there partitions to merge once per this interval, seconds")
+	fd.Duration("loop-interval", viper.GetDuration("daemon.loop-interval"), "Daemon will check if there partitions to merge once per this interval, duration")
```

Comment:
`The daemon will check if there partitions to merge once per this interval (default 1h0m0s)`

Adding the duration word here makes it confusing, imo